src/table/components/head/TableHeadWrapper.tsx

```diff
@@ -35,6 +35,7 @@ function TableHeadWrapper({
   const headRowRef = useRef<HTMLElement>();
   const isInteractionEnabled = !constraints.active && !selectionsAPI.isModal();
 
+  const [menuOpen, setMenuOpen] = useState(false);
```

Comment:
bit ocd here, but put the state with the other use things, above `isInteractionEnabled`

Reply:
What do you mean 'other use things'? [open, setOpen] ?